contracts/CTKNCrowdsale.sol

```diff
@@ -44,4 +49,68 @@ contract CTKNCrowdsale is CappedCrowdsale, RefundableCrowdsale, MintedCrowdsale
         dollarRate = _dollarRate;
         DollarRateSet(dollarRate);
     }
+
+    /**
+     *  @param addr The address to check for a refund balance
```

Comment:
It seems like it returns the balance from overpayment, so maybe comment can be
"The address to retrieve refund balance"